core/src/main/java/org/mskcc/cbio/portal/util/HeatMapLegend.java

```diff
@@ -80,15 +80,15 @@ static public String outputHeatMapLegend( OncoPrintGeneDisplaySpec allPossibleAl
       }
```

Comment:
The code in this file hasn't been changed in 9 years. I don't think we should change it.

Reply:
Reverted change to file